pkgs/os-specific/linux/firmware/rtl8723bs-firmware/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+let
+  rev = "c517f2bf8bcc3d57311252ea7cd49ae81466eead";
+in
+stdenv.mkDerivation rec {
+  name = "rtl8723bs-firmware-${rev}";
+
+  src = fetchFromGitHub {
```

Comment:
I suppose this could use the `src` attribute from the driver package, so that there is no need to update the git rev/sha256 in many places.